cleanup.js

```diff
@@ -39,6 +39,9 @@ try {
   const stopped_engine = await retryWithBackoff(async () => {
     return await firebolt.resourceManager.engine.getByName(stopped_engine_name);
   });
+  await retryWithBackoff(async () => {
+    await stopped_engine.stop();
+  });
```

Comment:
Let's add a check for if the engine is running - stop it. I think on engines v1 stopping an already stopped engine would fail.